recipes/fastlinkingnumbers/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "fastlinkingnumbers" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/antequ/fastlinkingnumbers/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b3393bffb8669ab8214b1b86ae8b1d48abd70ba021134a3f2a5e5fcfd641e7d9
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
+    - eigen
+    - libgomp
+  host:
+    - openmp
+    - gflags
+  run:
+    - openmp
+    - gflags
+
+test:
+  source_files:
+    - data
+    - results
+    - test.sh
+  commands:
+    - test -f $PREFIX/bin/verifycurves
+    - test -f $PREFIX/bin/obj2bcc
+    - ln -s $PREFIX/bin/verifycurves . && ln -s $PREFIX/bin/obj2bcc . && source test.sh
```

Comment:
This is a weird test. The binaries should work without putting them into the current directory.

Also, why do you need to source the shell script? Can't you simply execute it here?

Reply:
Maybe you want to patch `test.sh` so it calls the binaries on the PATH instead.

Reply:
The shell script from upstream is not executable. I'll work on a patch though.

Reply:
You could execute it with `sh test.sh` instead of sourcing it. That's what I meant.

Reply:
Changed to `sh test.sh`, still symlinking for now until I add a patch.